bundle/QueryType/SearchQueryType.php

```diff
@@ -27,38 +33,66 @@ public static function getName(): string
 
     protected function configureOptions(OptionsResolver $optionsResolver): void
     {
-        $optionsResolver->setRequired(['search_text', 'content_types', 'subtree']);
-
+        $optionsResolver->setRequired(['search_text', 'content_types', 'subtree', 'sort', 'order']);
         $optionsResolver->setAllowedTypes('search_text', 'string');
         $optionsResolver->setAllowedTypes('content_types', 'string[]');
+        $optionsResolver->setAllowedTypes('sort', 'string[]');
         $optionsResolver->setAllowedTypes('subtree', ['int', 'string']);
-
         $optionsResolver->setAllowedValues(
             'search_text',
             static fn (string $searchText): bool => trim($searchText) !== '',
         );
-
+        $optionsResolver->setAllowedValues(
+            'sort',
+            /** @var string[] $keys */
+            fn (array $keys): bool => $this->sortKeysAllowed($keys),
+        );
+        $optionsResolver->setAllowedValues(
+            'order',
+            static fn (string $order): bool => in_array($order, [Query::SORT_ASC, Query::SORT_DESC], true),
```

Comment:
Shouldn't `setAllowedValues` accept the array of allowed values, instead of a boolean?

From Syfmony docs:

```php
    public function configureOptions(OptionsResolver $resolver): void
    {
        $resolver->setDefault('transport', 'sendmail');
        $resolver->setAllowedValues('transport', ['sendmail', 'mail', 'smtp']);
    }
```

Reply:
Ah, disregard, it also accepts a closure with `bool` return type.